task/progress.go

```diff
@@ -41,7 +45,8 @@ func ReportProgress(ctx context.Context, lapi *api.Client, taskID string, size u
 				progressBucket(progress) == progressBucket(lastProgress) {
 				continue
 			}
-			if err := lapi.UpdateTaskStatus(taskID, "running", progress); err != nil {
+			scaledProgress := scaleProgress(progress, startFraction, endFraction)
```

Comment:
nit: could just iterate on the same variable like `progress = scaleProgress(progress, start, end)`

Reply:
Sounds good :)